drift/core/psmc.py

```diff
@@ -81,7 +85,7 @@ def _work_fisher_bias_m(self, mi):
             x = self.gen_sample(mi, n)
             qa[:, s:e] = self.q_estimator(mi, x)
 
-        ft = np.cov(qa)
+        # ft = np.cov(qa)
```

Comment:
Looks like this could be removed completely

Reply:
I agree, I will get rid of it... I just kept it because it was Richard's comments.